src/secrets/scraper.json

```diff
@@ -0,0 +1,20 @@
+{
+  "GitHub": [
+    {
+      "exclude": [],
+      "orgs": [
+        "cisagov"
+      ],
+      "public_only": true,
+      "repos": [],
+      "token": "ghp_IqIMN0ZH6z0wIEB4T9A2g4EHMy8Ji42q4HA5",
```

Comment:
Is this a real private token or a dummy value?

Reply:
That is the value that is shown in the [GitHub documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).

Reply:
Excellent.  Can you please add a comment to that effect so that no one else wonders about this in the future?

Reply:
I went ahead and replaced it with a value that's more obviously a placeholder in [d89852a](https://github.com/cisagov/code-gov-update/pull/30/commits/d89852a52074a0d69848ae4b0c499c611b13f963). The new value still conforms to the format of GitHub Personal Access Tokens while being more obviously a placeholder.

Reply:
Brilliant - thanks!